ops/roles/infra/files/mongod.conf

```diff
@@ -0,0 +1,23 @@
+# for documentation of all options, see:
+#   http://docs.mongodb.org/manual/reference/configuration-options/
+
+# Where and how to store data.
```

Comment:
I'm not sure of the added value of these comments.

If they stay here, they should have unified capitalisation and punctuation.

Reply:
It is the generated MongoDB configuration file, I just changed one line. Do you want I remove all comments ?

Reply:
However you want, my comment above still stands 🙂